app/views/users/show.html.erb

```diff
@@ -1 +1,26 @@
-<h1> Show users with particular ID </h1>
\ No newline at end of file
+<div>
+    <img src=<%= @user.photo%> alt="<%= @user.name%>'s photo">
+    <div>
+        <h1> <%= @user.name%> </h1>
+        <p> number of posts: <%= @user.posts_counter%></p>
+    </div>
+</div>
+<div>
+    <h2>Bio</h2>
+    <p> <%= @user.bio%> </p>
+</div>
+<div>
+    <a href="<%= @user.id %>/posts/"><h1> Posts </h1></a>
+    <div>
+        <% @user.posts.last(4).each do |post| %>
```

Comment:
- Please use the `last_3_posts` method you defined in the User model.

Reply:
Hi @afanimartin, I'll do it thank you 🙏